h_transport_materials/property_database/eurofer_97/eurofer_97.py

```diff
@@ -0,0 +1,183 @@
+import h_transport_materials as htm
+from h_transport_materials.property import Diffusivity, Solubility, Permeability
+from pathlib import Path
+
+import numpy as np
+
+u = htm.ureg
+
+aiello_permeability_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/aiello_2002/permeability.csv",
+    delimiter=",",
+)
+
+aiello_permeability_data_H = aiello_permeability_data[2:, :2]
```

Comment:
I've taken the bad habit of manually modifying the csv file (removing the X,Y,X,Y... and renaming the top header) to avoid doing this.

Maybe we should figure out a way to make devs life easier on this point

Reply:
I mean if we are going to recommend the use of webplotdigiser, could be useful to use the default of that. Otherwise yes we should have a standardised layout that works for 1 or multiple sets of data in the same csv file, and then specify this in the docs

Reply:
I'd like to have users:
- use WPD
- need minimal code for importing csv files in the script
- use the name of the column to avoid indexing errors

Reply:
I can open an issue for this and we can update the docs

Reply:
Sounds good!